craftassist/servermgr/app.py

```diff
@@ -149,54 +126,6 @@ def status():
     return json.dumps({"progress": 100, "ip": ip})
 
 
-@app.route("/matchmaker")
```

Comment:
Can we keep this somewhere in arxiv ? This was useful to do some matchmaking human - human experiments.